R/modules.R

```diff
@@ -155,6 +186,9 @@ is_arg_used <- function(modules, arg) {
 #'   `server` function.
 #' @param ui_args (named `list`) with additional arguments passed on to the
 #'   `ui` function.
+#' @param type (`character(1)`) The type of the class assigned to the final module. One of `"teal_module"`,
+#' `"teal_module_reporter"` or `"teal_module_landing"`. Modules of class `"teal_module_landing"` will not be wrapped
+#' into tabs in the `teal` application.
```

Comment:
```suggestion
#' @param type (`character(1)`) Class assigned to the resulting module. 
#'             All modules will have class `teal_module` (default) but one other class may be added. 
#'             Modules of class `"teal_module_landing"` will not be wrapped into tabs in the `teal` application.
```

Reply:
@chlebowa currently you either have a class `teal_module` or `teal_landing_module`. You can not have a module with two classes `c('teal_landing_module', 'teal_module')` as `inherits` in `drop/extract_module` will not work in determining `teal_module`

```r
x <- 5
class(x) <- c('teal_landing_module', 'teal_module')
inherits(x, 'teal_module')
TRUE
```

Reply:
Yes you can, consider the `if` and `else if`.

Reply:
I will try rewriting extract/drop_modules so they can handle modules with 2 classes